README.md

```diff
@@ -1,49 +1,85 @@
 # MLSQL-Deploy
+mlsql-deploy provides a CLI to deploy MLSQL Engine on K8S. The process consists of
+- mlsql-deploy CLI build
```

Comment:
Build mlsql-deploy from source

Reply:
Change is force-pushed, please click "latest change here" link